test/experiment_test.rb

```diff
@@ -450,4 +459,10 @@ def test_cleanup_without_redis
       experiment.cleanup
     end
   end
+
+  private
+
+  def redis
+    @redis ||= ::Redis.new(host: REDIS_HOST, port: REDIS_PORT)
```

Comment:
what redis is this connecting to?

Reply:
Some local instance. Previously, I've had to explicitly start one using `redis-server` to make redis dependent tests pass locally.

Reply:
Correct, the default one is running on localhost at the default port. This works on CI and locally when running redis-server, but redis setup is like this on master.